cmd/gctcli/commands.go

```diff
@@ -2805,3 +2806,112 @@ func clearScreen() error {
 		return cmd.Run()
 	}
 }
+
+const timeFormat = "2006-01-02 15:04:05"
+
+var startTime, endTime, order string
+var limit int
+
+var getAuditEventCommand = cli.Command{
+	Name:   "getauditevent",
+	Usage:  "gets audit events matching query parameters",
+	Action: getAuditEvent,
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:        "start, s",
+			Usage:       "start date to search",
+			Value:       time.Now().Add(-time.Hour).Format(timeFormat),
```

Comment:
One thing we'll need to be mindful of is server time vs user time, if they differ then the user will need to know that they'll have to specify the servers time